pkg/resource/testdata/TestStatefulSetBuilder/default_insecure_statefulset_in.yaml

```diff
@@ -31,7 +31,7 @@ spec:
   grpcPort: 26258
   httpPort: 8080
   image:
-    name: cockroachdb/cockroach:v20.2.7
+    name: cockroachdb/cockroach:v21.1.0
```

Comment:
We should make sure that this is `v21.1.1` or higher

Reply:
The design document suggest the logging api is supported `21.1.x` version and the lowest version is `v21.1.0`. Is there any specific reason we want to exclude `v21.1.0` and start supporting from `v21.1.1`?